include/package-ipkg.mk

```diff
@@ -102,6 +102,7 @@ ifeq ($(DUMP),)
     ABIV_$(1):=$(call FormatABISuffix,$(1),$(ABI_VERSION))
     PDIR_$(1):=$(call FeedPackageDir,$(1))
     IPKG_$(1):=$$(PDIR_$(1))/$(1)$$(ABIV_$(1))_$(VERSION)_$(PKGARCH).ipk
+    DBGINFO_$(1):=$$(PDIR_$(1))/$(1)$$(ABIV_$(1))_$(VERSION)_$(PKGARCH).debuginfo
```

Comment:
Can't we throw it all in a single debuginfo folder like we do it for kmods right now? This way we wouldn't need a subdomain or anything.

Reply:
A subdomain shouldn't be required either way (`debuginfod -F "bin/packages/`) but i guess, right now, it's saves the time looking at uninteresting files.
Storing the debuginfo in `bin/packages/` (`debuginfod -F "bin/packages/`)  was deliberate choice. `${arch}_${cpu_type}/${feed}/` is just the way files are organised in there.

debuginfod also parses packages and could also provide sources (probably from tar-files in `bin/packages/..`).
Openwrt is eyeballing a switch to the .apk package format. apk-files can probably be ready somehow using `-Z '.apk' which would turn all the 'uninteresting' package files into interesting ones (containing binaries and/or source code).

I'm partial to keeping it the way it is.

Reply:
> Can't we throw it all in a single debuginfo folder like we do it for kmods right now?

Yes, good point,  its a mess currently. Maybe single `bin/debuginfo` folder would work?

>  This way we wouldn't need a subdomain or anything.

Ok, I'm probably missing something, but how would you serve it otherwise? You need `debuginfod` server for the clients, there is a web API.